dashboard/app/views/pd/workshop_mailer/teacher_enrollment_receipt.html.haml

```diff
@@ -10,8 +10,7 @@
 - if @workshop.local_summer?
   %p
     Thanks for enrolling in
-    = @workshop.organizer.name
-    ’s 5-day Summer workshop on the Code.org CS Principles curriculum.
+    = "#{@workshop.organizer.name}’s 5-day Summer workshop on the Code.org CS Principles curriculum."
```

Comment:
Proper Nouns ending in `s` should have the apostrophe at the end of the word, not have an appending s. I'm guessing this would be
```#{@workshop.organizer.name}'#{@workshop.organizer.name.ends_with?(s) ? 's' : ''}```